ImportService/lib/handlers/import-file-parser/import-file-parser.ts

```diff
@@ -0,0 +1,81 @@
+const csv = require("csv-parser");
+
+import { FIXME } from './../../../../ProductService/models/index';
+
+import { CopyObjectCommand, DeleteObjectCommand, GetObjectCommand, S3Client } from "@aws-sdk/client-s3";
+import { Product } from 'aws-cdk-lib/aws-servicecatalog';
```

Comment:
Seems to be unrelevant to the project, please refer to docs to read about `Product` in AWS ecosystem (https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_servicecatalog-readme.html)

Reply:
hmm auto import...